src/components/AppSummary/AppSummary.columns.js

```diff
@@ -49,8 +69,17 @@ export const getFeesColumns = ({
     width: 140,
     renderer: () => (
       <Cell>
```

Comment:
this particular Cell could be extracted to a dedicated custom Cell component in order to be reused just by passing `isLoading` and `fee` values to it